packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -60,29 +56,27 @@ export interface DotnetPluginConfig {
 }
 
 export class DotnetPluginImpl implements PluginImpl {
-  config: DotnetPluginConfig = {};
-
-  private syncConfigFromContext(ctx: PluginContext): void {
```

Comment:
Nit: Good change to remove the data member of config and sync/generate new config in runtime by needs.
It will get rid of the side-effect risks when plugins store their own states.